calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -505,3 +515,40 @@ func (rh resourceHelper) GetTableTemplate(headings []string) (string, error) {
 
 	return buf.String(), nil
 }
+
+// overwriteMetadata compares the Metadata for a ResourceObject and warns if anything was
+// changed that will not be changed as a part of the update. Any discrepancies to the metadata
+// will be changed to match the metadata available in the stored object except for Labels and
+// Annotations.
+func overwriteMetadata(stored, changed ResourceObject) ResourceObject {
```

Comment:
How about `old, new` rather than `stored, changed` - it's a little more standard.